src/main/java/lt/vu/web/api/v1/controller/user/UpdateUserInfoController.java

```diff
@@ -0,0 +1,78 @@
+package lt.vu.web.api.v1.controller.user;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.parameters.RequestBody;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import lt.vu.application.exception.BadRequestException;
+import lt.vu.application.exception.NotFoundException;
+import lt.vu.application.security.model.Token;
+import lt.vu.application.user.service.UpdateUserInfoService;
+import lt.vu.infrastructure.security.Authorized;
+import lt.vu.persistence.orm.repository.UserRepository;
+import lt.vu.web.api.v1.controller.security.CurrentUserAwareController;
+import lt.vu.web.api.v1.dto.security.GetTokenDTO;
+import lt.vu.web.api.v1.dto.user.GetUserDTO;
+import lt.vu.web.api.v1.dto.user.UpdateUserDTO;
+import lt.vu.web.api.v1.dto.userInfo.PostUserInfoDTO;
+import lt.vu.web.api.v1.exception.ExceptionDTO;
+
+import javax.enterprise.context.RequestScoped;
+import javax.inject.Inject;
+import javax.transaction.Transactional;
+import javax.validation.Valid;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Path("/user/details")
+@RequestScoped
+public class UpdateUserInfoController extends CurrentUserAwareController {
+
+    @Inject
+    private UpdateUserInfoService updateUserInfoService;
+
+    @PUT
+    @Path("/")
+    @Authorized
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    @Transactional
+    @Operation(
+            summary = "Change user's data",
```

Comment:
I know IntelliJ does this, but for consistency, we have 4 spaces (1 tab) indentation in other files.

Reply:
Yes.. Didn't notice that :D Linter would be great :D